docs/TESTING.md

```diff
@@ -1,53 +1,60 @@
 ### Test instructions (for developers):
 
-This is a rough sketch, some more background is found in [JM wiki](https://github.com/Joinmarket-Org/joinmarket/wiki/Testing)
+Work in your `jmvenv` virtualenv as for all Joinmarket work. Make sure to have [bitcoind](https://bitcoin.org/en/full-node) 0.17 or newer installed. Also need miniircd installed to the root (i.e. in your `joinmarket-clientserver` directory):
 
-Make sure to have [bitcoind](https://bitcoin.org/en/full-node) 0.17 or newer installed. Also need miniircd installed to the root (i.e. in your `joinmarket-clientserver` directory):
+    (jmvenv)$ cd /path/to/joinmarket-clientserver
+    (jmvenv)$ git clone https://github.com/Joinmarket-Org/miniircd
 
-    cd ~/joinmarket-clientserver
-    git clone https://github.com/Joinmarket-Org/miniircd
+Install the test requirements:
 
-Install the test requirements (still in your virtualenv as mentioned above):
+    (jmvenv)$ pip install -r requirements/testing.txt
 
-    pip install -r requirements/testing.txt
+#### Running the test suite.
 
-Running the test suite should be done something like (advisable to wipe ~/.bitcoin/regtest first):
+Have a `bitcoin.conf` ready in some location, whose contents only need to be:
 
-    pytest --btcconf=/path/to/bitcoin.conf --btcroot=/path/to/bitcoin/bin/ --btcpwd=whatever --nirc=2 --ignore test/test_full_coinjoin.py -p no:warnings
-    
-(you'll first want to copy the regtest_joinmarket.cfg file from the test/ directory to the root directory,
-this file will need minor edits for your btc configuration).
+```
+rpcuser=bitcoinrpc
+rpcpassword=123456abcdef
+fallbackfee=0.0002
+```
 
-### Running tests of sendpayment and tumbler (including with malicious makers)
+(any random password is fine of course). It is also advisable to wipe ~/.bitcoin/regtest first, in case it gets large and slow to process.
 
-(As well as the below, there is a rudimentary "single, automatic, end-to-end coinjoin test" in the file test/test_full_coinjoin.py. Recommended to run it
-with -s to see log output; what's "rudimentary" here is that errors may simply result in the process hanging, so you'll want to investigate in that case.
-Use command line:
+Then copy the `regtest_joinmarket.cfg` file from the `test/` directory to the root directory, you probably won't need to change anything here except perhaps the above password, and the `native` setting if you're doing bech32 wallet tests.
```

Comment:
Three things:
1. It seems to me `regtest_joinmarket.cfg` needs to be renamed to `joinmarket.cfg`
2. By root directory, maybe we should clarify we mean the root of joinmarket, not the root of the user?
3. The way I'm doing things right now, the tests will **only** work with the `native` setting on. Let me know if this is an issue.

Reply:
1 and 2, yep. 3: ? I don't know what is causing that, for sure that isn't the normal case, maybe it's just in your PR or something.

Re: EDIT, well ... not really, if a person is running current master and wants to test native behaviour, they have to explicitly set it in the config file. The settings in configure.py are the defaults, and are overriden if they exist in joinmarket.cfg.

Reply:
1,2 done in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/29875b4a2ae151afbc3223271c8c7d5c325f0e46

Reply:
I'm taking a look at my work and will get back to you.